core/model/modx/processors/element/propertyset/updatefromelement.class.php

```diff
@@ -9,44 +10,39 @@
 
 class modPropertySetUpdateFromElementProcessor extends modPropertySetUpdateProcessor {
     public $languageTopics = array('propertyset', 'category', 'element');
-
-    /** @var modPropertySet|null */
-    public $object;
```

Comment:
@Fi1osof We told you how to correct it, just do what @theboxer said to do and you'll be fine.